COMETwebapp/Extensions/ThingExtensions.cs

```diff
@@ -87,7 +87,7 @@ public static IEnumerable<ParameterValueSetBase> QueryParameterValueSetBase(this
             return valueSets.DistinctBy(x => x.Iid);
         }
 
-        /// <summary>rie
+        /// <summary>
```

Comment:
Please add a ticket to move all of these convenience methods to the COMET-SDK so they are also available for other implementations. I would not be surprised that they may already exist

Reply:
https://github.com/RHEAGROUP/COMET-SDK-Community-Edition/issues/251